clarity_ext/automated_tests/test_cases/dilute_tests/hamilton_tests.py

```diff
@@ -0,0 +1,52 @@
+# Unit test the output of the dilution script, generating driver file for Hamilton
+# Using the utility in dilute_filer_reader
+
+import unittest
+import os
+import inspect
+from ...utility.dilute_filer_reader.hamilton_driver_file_reader import HamiltonReader as FileReader
+from ...utility.dilute_filer_reader.hamilton_driver_file_reader import HamiltonColumnReference as ColumnRef
+from ....dilute_epp import perform_dilution
+
+TEST_PROCESS_URI = "https://lims-staging.snpseq.medsci.uu.se/api/v2/processes/24-3251"
+SAMPLE1 = "EdvardsProv1Updated"
+SAMPLE2 = "EdvardsProv3"
+SAMPLE3 = "EdvardsProv2"
+SAMPLE4 = "EdvardsProv4"
+
+
+class HamiltonTests(unittest.TestCase):
+
+    def setUp(self):
+        driver_file_contents = perform_dilution.create_driver_file(TEST_PROCESS_URI, None)
```

Comment:
We can look into if the framework stuff in the clarity-ext project wouldn't help with this.

It provides a different take on integration tests, where the tests are executed once and then again on each build, comparing the results as a whole. They have the extra advantage that the process step does not need to exist anymore, since they use cached requests/responses.